modules/ui_extra_networks.py

```diff
@@ -85,6 +85,30 @@ def get_metadata(page: str = "", item: str = ""):
 
     return HTMLResponse(metadata_html)
 
+def item_filter(key: str, item: dict) -> bool:
+    return key in item.get('search_term', '')
+
+def get_page_query_model(request: Request, model_type:str, page: int, search_value: str, page_size: int):
+    from starlette.responses import JSONResponse
+    
+    model_list = []
+    count = 0
+    
+    def item_filter(item: dict) -> bool:

```

Comment:
You already has a `item_filter` defined outside the scope of this function, and you defined it again, is there a particular reason?